### PR TITLE
Conf and test update

### PR DESCRIPTION
**Pull request by pbailis**
Minor change: default OI-ratio for streaming and batch is 3, but lower it to 1 for the GUI.
Minor change: exercise sanity checking and environment loading in tests.

### CODE REVIEW

**Comment by pbailis** [comment]
Can someone look this over?

File: src/test/java/macrobase/analysis/BatchAnalyzerTest.java [hunk 1]
```diff
@@ -44,6 +44,9 @@ public void testMADAnalyzer() throws Exception {
         conf.set(MacroBaseConf.DATA_LOADER_TYPE, MacroBaseConf.DataLoaderType.CSV_LOADER);
         conf.set(MacroBaseConf.CSV_INPUT_FILE, "src/test/resources/data/simple.csv");
 
+        conf.loadSystemProperties();
+        conf.sanityCheckBatch();
```
Comment (deepakn94) [inline src/test/java/macrobase/analysis/BatchAnalyzerTest.java]: This method wasn't added in this set of commits?

Comment (pbailis) [inline src/test/java/macrobase/analysis/BatchAnalyzerTest.java]: no, it existed before

Comment (deepakn94) [inline src/test/java/macrobase/analysis/BatchAnalyzerTest.java]: Ok, looks good.

**Comment by deepakn94** [comment]
LGTM.

**Comment by pbailis** [comment]
Thanks!

**Pull request merged**